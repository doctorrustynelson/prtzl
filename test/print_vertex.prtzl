Vertex v = <+ "vertex" +>;
print_vertex(v);