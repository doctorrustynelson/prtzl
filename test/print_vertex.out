vertex: 0 edge(s) in 0 edge(s) out
